twins/swagger.yaml

```diff
@@ -0,0 +1,336 @@
+swagger: '2.0'
+info:
+  title: Mainflux twins service
+  description: HTTP API for managing digital twins and their states.
+  version: '1.0.0'
+consumes:
+  - 'application/json'
+produces:
+  - 'application/json'
+paths:
+  /twins:
+    post:
+      summary: Adds new twin
+      description: |
+        Adds new twin to the list of twins owned by user identified using
+        the provided access token.
+      tags:
+        - twins
+      parameters:
+        - $ref: '#/parameters/Authorization'
+        - name: twin
+          description: JSON-formatted document describing the new twin.
+          in: body
+          schema:
+            $ref: '#/definitions/TwinReq'
+          required: true
+      responses:
+        201:
+          description: Twin registered.
+          headers:
+            Location:
+              type: string
+              description: Created twin's relative URL (i.e. /twins/{twinID}).
+        400:
+          description: Failed due to malformed JSON.
+        403:
+          description: Missing or invalid access token provided.
+        415:
+          description: Missing or invalid content type.
+        500:
+          $ref: '#/responses/ServiceError'
+
+    get:
+      summary: Retrieves managed twins
+      description: |
+        Retrieves a list of managed twins. Due to performance concerns, data
+        is retrieved in subsets.
+      tags:
+        - twins
+      parameters:
+        - $ref: '#/parameters/Authorization'
+        - $ref: '#/parameters/Limit'
+        - $ref: '#/parameters/Offset'
+        - $ref: '#/parameters/Metadata'
```

Comment:
looks like `name` is missing here

Reply:
No, everything is ok. You can check it by pasting it here https://editor.swagger.io/

BTW, those are parameter **references**, not comments, and parameters, in this case are defined in `parameters` secton.

Reply:
https://github.com/mainflux/mainflux/blob/master/twins/api/http/transport.go#L167